app/src/main/java/org/stypox/dicio/App.kt

```diff
@@ -0,0 +1,23 @@
+package org.stypox.dicio
+
+import android.app.Application
+import androidx.core.app.NotificationChannelCompat
+import androidx.core.app.NotificationManagerCompat
+
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        val notificationManager = NotificationManagerCompat.from(this)
```

Comment:
This can be simplified, no need for a val assignment

Reply:
Right, didn't catch this